assets/js/nps.js

```diff
@@ -0,0 +1,41 @@
+function fNpsApi() {
+    // var searchTerm = document.querySelector('#searchTerm').value;
+
+    var searchTerm = 'CA';
+    var sNpsApi = 'https://developer.nps.gov/api/v1/campgrounds?' +
+        'stateCode=' +
+        searchTerm +
+        '&limit=500' +
```

Comment:
Why did you decide on a limit of 500? Is this just to guarantee we get every result?